sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/PushDownOperatorsToDataSource.scala

```diff
@@ -26,7 +26,7 @@ object PushDownOperatorsToDataSource extends Rule[LogicalPlan] {
   override def apply(
       plan: LogicalPlan): LogicalPlan = plan transformUp {
     // PhysicalOperation guarantees that filters are deterministic; no need to check
-    case PhysicalOperation(project, newFilters, relation : DataSourceV2Relation) =>
+    case PhysicalOperation(project, newFilters, relation: DataSourceV2Relation) =>
```

Comment:
While I'd rather not have this space there either (looks like an accident), there are no other changes to this file and this "fix" is not necessary. The risk of this causing commit conflicts outweighs the benefit of conforming to style so it should be removed.